src/main/webapp/app/overview/course-conversations/dialogs/conversation-detail-dialog/tabs/conversation-info/conversation-info.component.ts

```diff
@@ -171,7 +171,7 @@ export class ConversationInfoComponent implements OnInit, OnDestroy {
 
     private updateGroupChat(groupChat: GroupChatDTO, propertyName: string, updateValue: string) {
         const updateDTO = new GroupChatDTO();
-        updateDTO[propertyName] = updateValue;
+        (updateDTO as any)[propertyName] = updateValue;
```

Comment:
can we avoid `as any` casts please? The whole method `updateGroupChat` does not really make sense to me to be honest. Why should we change attributes in a specific method?